test.c

```diff
@@ -1839,6 +1866,32 @@ response_status_cb (http_parser *p, const char *buf, size_t len)
   return 0;
 }
 
+int
+chunk_header_cb (http_parser *p)
+{
+  assert(p == parser);
+  if (p->content_length == 0) {
+    // Terminating chunk.  Don't increment num_chunks in this case.
```

Comment:
Yep, it just feels a bit odd that you have different number of complete and seen chunks in test declarations :)